readme.md

```diff
@@ -183,7 +192,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 
 * **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route.
 
-* **Your Answer:**
+* **Your Answer:** middleware is something that acts as a wrapper and exposes many methods within app for easy access. Without middleware we can still build the entire server but just that the code will tend to be overly verbose. middleware acts as a translation unit and makes life easier by providing useful wrappers
```

Comment:
Middleware is the code/application/module that communicates between two systems, such as between servers and databases. The code above is different than a route because it is always run whenever the application gets to these lines.